eventActions/tosReminderAction.js

```diff
@@ -0,0 +1,63 @@
+const Discord = require('discord.js')
+table = require('../databaseFiles/tosReminderTable');
+
+const addToDatabase = async function(user,joinTime,reminded=false){
+    table.sync(/*{ force: true }*/).then(async ()=>{
+        await table.create({
+            user_id: user.id,
+            joinTime: joinTime,
+            reminded: reminded
+        }).catch(err =>{
+            console.error('Tosreminder error: ' + err )
+        })
+    })
+}
+const removeFromDatabase = async function(user_id){
```

Comment:
The `addToDatabase` function takes a user, while `removeFromDatabase` takes a user's id. For consistency, these should both be the same - I think it'd be better to pass the user for both.